README.md

```diff
@@ -3,6 +3,9 @@
 [![Shadow Tests](https://github.com/shadow/shadow/actions/workflows/run_tests.yml/badge.svg?branch=main&event=push)](https://github.com/shadow/shadow/actions/workflows/run_tests.yml?query=branch:main+event:push)
 [![Tor Tests](https://github.com/shadow/shadow/actions/workflows/run_tor.yml/badge.svg?branch=main&event=push)](https://github.com/shadow/shadow/actions/workflows/run_tor.yml?query=branch:main+event:push)
 
+[Documentation](https://shadow.github.io/docs/guide) +
```

Comment:
Optional stylistic nit - I'd lean towards putting these each on their own line.

IIRC the way to force a newline in GH's flavor of markdown is with a blank space at the end of the line.

An unordered list would also work.

Reply:
Changed them to be on separate lines.